ast/compile.go

```diff
@@ -902,7 +902,7 @@ func parseSchema(schema interface{}) (types.Type, error) {
 			return types.N, nil
 
 		} else if subSchema.Types.Contains("object") {
-			if subSchema.PropertiesChildren != nil && len(subSchema.PropertiesChildren) > 0 {
+			if len(subSchema.PropertiesChildren) > 0 {
```

Comment:
Feels like there should be a linter ruler for this :)

Reply:
I think the gosimple linter has that... I don't know how/if that's surfaced through golangci-lint, though, it seems like we've enabled the gosimple linter there...

Reply:
@willbeason do you know? :)